api/models/roomDao.js

```diff
@@ -77,7 +77,70 @@ const getRoomByDetail = async (roomId) => {
   }
 };
 
+const addRoomToWishList = async (userId, roomId) => {
+  try {
+    return await mysqlDatabase.query(
+      `
+      INSERT INTO wish_lists (
+        user_id,
+        room_id
+      ) VALUES (?, ?)
+      `,
+      [userId, roomId]
+    );
+  } catch (err) {
+    console.error(err.stack);
+    detectError('DATABASE_ERROR', 500);
```

Comment:
상태코드 500말고 400으로 해주세요. 
500이면 에러처리가 안 된 것으로 간주 될 수 있습니다.
```suggestion
    detectError('DATABASE_ERROR', 400);
```